.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,106 +1,117 @@
 
-        *Before submitting your issue*, please ensure that:
-
-        1. You've searched the [existing issues](https://github.com/Shopify/shopify-cli/issues) to see if someone else has already submitted the same thing. (Feel free to add any additional information in a comment!)
-        2. You've given as much detail as reasonably possible
-        3. You're working with the [latest CLI version](https://github.com/Shopify/shopify-cli/releases)
-
-        This is important because:
-
-        1. We will notice and prioritize 1 popular issue more quickly than many small duplicates
-        2. More details = faster, higher-quality response
-        3. We might have fixed your issue already!
-
+        **Before submitting your issue**, please confirm:
+      options:
+        - label: I've searched the [existing issues](https://github.com/Shopify/shopify-cli/issues) to see if someone else has already submitted the same thing. (Feel free to add any additional information in a comment!)
+          required: true
+        - label: I've given as much detail as reasonably possible
+          required: true
+        - label: I've tested with the [latest CLI version](https://github.com/Shopify/shopify-cli/releases)
+          required: true
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: '## General Questions'
   - type: dropdown
-    id: app-type
+    id: app_type
     attributes:
-      label: What type of project are you working with?
+      label: What type of project are you working on?
       options:
         - app
         - extension
         - script
         - theme
+        - other
     validations:
       required: true
-  - type: textarea
-    id: issue-summary
+  - type: input
+    id: issue_summary
     attributes:
       label: Issue summary
       description: Write a short description of the issue here ↓
       placeholder: Tell us what you see!
     validations:
       required: true
   - type: textarea
-    id: expected-behavior
+    id: expected_behavior
     attributes:
       label: Expected behavior
       description: What do you think should have happened?
       placeholder: It should have worked
     validations:
       required: true
   - type: textarea
-    id: actual-behavior
+    id: actual_behavior
     attributes:
       label: Actual behavior
       description: What actually happened?
       placeholder: It didn't work
     validations:
       required: true
   - type: textarea
-    id: stack-trace
+    id: stack_trace
     attributes:
       label: Stack trace
       description: Paste in a stack trace if you've got one!
       render: shell
     validations:
       required: false
   - type: textarea
-    id: repro-steps
+    id: repro_steps
     attributes:
       label: Reproduction steps
-      description: Let us know the exact steps required to reproduce the error. The more detail, the better! (markdown accepted)
```

Comment:
All fields are markdown accepted and give the :octocat: editor so this seemed unnecessary.